### ISSUE TITLE
Thaumcraft aspects in items search (enhancement request)

### ISSUE THREAD

**Issue by MacChuck** [mezz/JEIAddons#8]
Would it be possible to make it so you can search JEI for a Thaumcraft aspect (say, Ordo), and click on it to get a list of what items are registered as having ordo in them? I know you can look in the Thauminomocon, but sometimes it is hard to tell what an item is when it is scrolling by.

**Comment by Prospector** [comment]
Please! It's annoying having to keep referencing the thaumonomicon.

**Comment by mezz** [comment]
Yeah the Thaumonomicon is pretty bad for this, and I run into the issue often when figuring out crucible recipes. I need to think of a way of adding this feature without making it a horrible buggy hack though. I'm pretty sure I will have to create a fake item for the aspects.

**Comment by Prospector** [comment]
You could probably ask azanor if there's an easier way.

**Comment by MacChuck** [comment]
Any idea how Thaumcraft NEI plugin does it? (http://www.curse.com/mc-mods/minecraft/225095-thaumcraft-nei-plugin) It may be hacky, though, I haven't looked at the code for it.

**Comment by TehNut** [comment]
The NEI plugin creates an unobtainable item for each aspect and uses a custom recipe handler.

**Comment by AljoschaMeyer** [comment]
Came to this repo to suggest exactly this.

How would a good UI look for this? Add fake items to the normal JEI interface, and also make the recipes accessible from the essentia smelteries?

Or (not xor) maybe an itemfilter for the search, just like "@Modname" but for aspects. Not "discoverable" for users, but very convenient if you know about it.
